mne/io/eeglab/eeglab.py

```diff
@@ -347,8 +347,8 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
         info['chs'].append(stim_chan)
         info._update_redundant()
 
-        events = _read_eeglab_events(eeg, event_id=event_id,
-                                     event_id_func=event_id_func)
+        events = read_events_eeglab(eeg, event_id=event_id,
+                                    event_id_func=event_id_func)
```

Comment:
I don't understand how that'd work. Called without additional parameters, read_raw_eeglab and read_events_eeglab will perform a many-to-one mapping: strip every key of its non-numeric parts, and use that as the value. So it will do stuff like:

```
'S101' -> 101
'R101' -> 101
'101' -> 101
101 -> 101
'square' -> (dropped)
```

To recover `square`, you'd have to use event_id. To have values distinguishing between 'S101' and 'R101', you'd have to use event_id too. So in any unambiguous mapping, the user will have to construct the event_id themselves before calling the function in the first place.